src/shogun/mathematics/linalg/internal/modules/SpecialPurpose.h

```diff
@@ -67,15 +69,15 @@ void rectified_linear(Matrix A, Matrix result)
 	implementation::special_purpose::rectified_linear<backend, Matrix>::compute(A, result);
 }
 
-/** Performs the operation C(i,j) = C(i,j) * (A(i,j)!=0) for all i and j*/ 
+/** Performs the operation C(i,j) = C(i,j) * (A(i,j)!=0) for all i and j*/
 template <Backend backend=linalg_traits<Core>::backend,class Matrix>
 void multiply_by_rectified_linear_derivative(Matrix A, Matrix C)
 {
 	implementation::special_purpose::multiply_by_rectified_linear_derivative<backend, Matrix>::compute(A, C);
 }
 
-/** Applies the softmax function inplace to a matrix. The softmax function is 
- * defined as \f$ f(A[i,j]) = \frac{exp(A[i,j])}{\sum_i exp(A[i,j])} \f$ 
+/** Applies the softmax function inplace to a matrix. The softmax function is
+ * defined as \f$ f(A[i,j]) = \frac{exp(A[i,j])}{\sum_i exp(A[i,j])} \f$
  */
 template <Backend backend=linalg_traits<Core>::backend,class Matrix>
 void softmax(Matrix A)
```

Comment:
Yes, I may use in near future once the four FITC methods are implemented.
There are two additional inference methods (all about Emti's KL dual inference method) to be added in the multiclass logistic GPs.  I will do it once I get time.